python/lsst/ip/isr/overscan.py

```diff
@@ -131,6 +152,9 @@ def run(self, ampImage, overscanImage):
                 overscanArray[:, :] = overscanValue[:, np.newaxis]
                 if maskSuspect:
                     maskSuspect.getArray()[maskArray, :] |= ampImage.getMask().getPlaneBitMask("SUSPECT")
+        else:
+            raise pexExcept.Exception('%s : %s an invalid overscan type' %
```

Comment:
Also in python we really want to raise python exceptions not pex_exception exceptions. Those exist so that C++ code can raise python-style exceptions.

Reply:
Changed to RuntimeError.